src/libs/CardUtils.ts

```diff
@@ -398,6 +407,17 @@ function checkIfNewFeedConnected(prevFeedsData: CompanyFeeds, currentFeedsData:
     };
 }
 
+function getAllCardsForWorkspace(workspaceAccountID: number): CardList {
+    const cards = {};
+    for (const [key, values] of Object.entries(allWorkspaceCards ?? {})) {
+        if (key.includes(workspaceAccountID.toString()) && values) {
+            const {cardList, ...rest} = values;
```

Comment:
Can you please add a comment that in case of direct feeds, the list of unassigned cards yet is returned in the cardList property in the collection?